src/js/controllers/mapController.ts

```diff
@@ -896,7 +906,7 @@ export class MapController {
       layer = this._map
         ?.findLayerById(parentID)
         //@ts-ignore -- sublayers exist
-        ?.allSublayers.items.find((sub: any) => sub.id === layerID);
+        ?.allSublayers.items.find((sub: any) => sub.id == layerID);
```

Comment:
is there a reason why you removed `===` to `==` ? 
The difference between these two is that the abstract equality operator doesn’t care about the data types, whereas the strict equality operator does.

Reply:
I removed because the `layerID` was coming across as a string but was looking for a number to match with the `sub.id`.

Reply:
@davidlstarr  yes because if you compare 
```js
 '1' == 1  output true // which in reality these two values are not the same
```

you can just do
```js
Number(sub.id) === Number(layerID)
```

Reply:
cool I will make the change